airbyte-integrations/connectors/source-kyve/source_kyve/schemas/bitcoin/block.json

```diff
@@ -0,0 +1,83 @@
+{
+  "$schema" : "http://json-schema.org/draft-04/schema#",
+  "type" : "object",
+  "properties" : {
+    "hash" : {
+      "type" : "string"
+    },
+    "height" : {
+      "type" : "integer"
+    },
+    "version" : {
+      "type" : "integer"
+    },
+    "versionHex" : {
+      "type" : "string"
+    },
+    "merkleroot" : {
+      "type" : "string"
+    },
+    "time" : {
+      "type" : "integer"
+    },
```

Comment:
For each of the fields in the schema files, can you modify it so that they are nullable? This will help make the connector less likely to break with schema changes.

Reply:
Resolved